src/api/index.test.js

```diff
@@ -56,22 +56,22 @@ describe('getAPIRoot', () => {
 
 describe('getAPI', () => {
   it('returns a URI containing the given type', () => {
-    const uri = getAPI('pipeline');
-    expect(uri).toContain('pipeline');
+    const uri = getAPI('pipelines');
```

Comment:
I don't see any update to the implementation in src/api/index.js. The unit tests should have caused the build to fail.